example/models/city/v1/schema_test.go

```diff
@@ -23,7 +22,7 @@ func TestSchemaAttractions(t *testing.T) {
 	require.NoError(t, err)
 	for _, line := range schema {
 
-		fmt.Printf("%s \n", line)
+		//fmt.Printf("%s \n", line)
```

Comment:
should be removed?

Reply:
I think these prints are fine to keep in the tests